nucleus/grizzly/config/src/test/java/org/glassfish/grizzly/config/GrizzlyConfigTest.java

```diff
@@ -310,10 +312,15 @@ public void testNetworkAddressValidator() {
         assertFalse(validator.isValid("${SOME_PROP", null));
         assertFalse(validator.isValid("{SOME_PROP}", null));
         assertTrue(validator.isValid("127.0.0.1", null));
-        assertFalse(validator.isValid("1271.2.1.3", null));
+        if (!onMacOsX()) assertFalse(validator.isValid("1271.2.1.3", null));
```

Comment:
I ran the GF build against master on my mac and this test did not fail.

Reply:
You took out my conditional? What version of OSX? It fails for me on both Mojave and Catalina.

Reply:
10.14.6 (Mojave) ; JDK 1.8.0_191

Reply:
Hi @russgold 
Could you grep your (on the _machines_ that resolves this `1271.2.1.3`) `/etc/hosts` for `1271.2.1.3`?
Or nslookup it in your (used by _machines_) DNS?
Because for me `validator.isValid`
- returns **false** when I have **no** mapping for 1271.2.1.3 in hosts nor in DNS - (because it isn't IP AND can't be resolved to name)
- and returns **true** with any other naming service resolving this to given IP (as it is not proper IP address BUT resolvable hostname) .

And in https://www.eclipse.org/lists/glassfish-dev/msg00232.html I saw that in your case `1271.1.2.3` was resolved to `92.242.140.21`, so if it is valid and good for `ping`, why you'd expect it be invalid in Java?

Finally - this test WILL fail on other systems (at least on linux non-macos (fedora 31)) should the **name** 1271.2.1.3 be mapped (hosts or DNS, probably LDAP or NIS, etc. as well), as it depends on external system out of control of the test. Not good test IMO.

Reply:
$ nslookup 1271.2.1.3
Server:		192.168.1.1
Address:	192.168.1.1#53

Non-authoritative answer:
Name:	1271.2.1.3
Address: 92.242.140.21

I agree that it is a bad test, as it is actually just testing the behavior of InetAddress.getByName() more than our code. It should probably be fixed simply to ensure that it calls that method, and trust the implementation to work.

Reply:
> It should probably be fixed simply to ensure that it calls that method, 

I don't know. But I'd like to see condition in 
https://github.com/javaee/glassfish/blob/7fd391c0a0737065451079d5868b504ebe8b4989/nucleus/grizzly/config/src/test/java/org/glassfish/grizzly/config/GrizzlyConfigTest.java#L315
and then
https://github.com/javaee/glassfish/blob/7fd391c0a0737065451079d5868b504ebe8b4989/nucleus/grizzly/config/src/test/java/org/glassfish/grizzly/config/GrizzlyConfigTest.java#L320-L323 removed, as it with no reason blames mac os :grin: